service/controller.go

```diff
@@ -873,15 +891,22 @@ func (s *service) ControllerPublishVolume(
 		fsID := getFilesystemIDFromCsiVolumeID(csiVolID)
 		fs, err := s.getFilesystemByID(fsID, systemID)
 		if err != nil {
-			if strings.EqualFold(err.Error(), sioGatewayVolumeNotFound) || strings.Contains(err.Error(), "must be a hexadecimal number") {
+			if strings.EqualFold(err.Error(), sioGatewayFilesystemNotFound) || strings.Contains(err.Error(), "must be a hexadecimal number") {
 				return nil, status.Error(codes.NotFound,
-					"volume not found")
+					"filesystem not found")
 			}
 			return nil, status.Errorf(codes.Internal,
-				"failure checking volume status before controller publish: %s",
+				"failure checking filesystem status before controller publish: %s",
```

Comment:
update the message wrt to volume, to keep the consistency throughout.

Reply:
done